control-plane-operator/controllers/hostedcontrolplane/assets/olm/catalog-certified-operators.yaml

```diff
@@ -1,15 +0,0 @@
-apiVersion: "operators.coreos.com/v1alpha1"
```

Comment:
Please let's use imperative mood in the commit subject (action) and elaborate the motivation in the commit description, i.e git ci -m"Imperative mood action" -m"Why"